src/components/proximities/proximities-section.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faUtensils,
+  faTree,
+  faSchool,
+  faUniversity,
+  faHospital,
+  faStore,
+  IconDefinition,
+} from '@fortawesome/free-solid-svg-icons';
+import './proximities-section.css';
+import { H2 } from '../../styles/typography/h2';
+import { Body2 } from '../../styles/typography/body-2';
+import { colors } from '../../styles/colors';
+import { H3 } from '../../styles/typography/h3';
+import { Separator } from '../separator/separator';
+import { ProximitiesMap } from './proximities-map';
+
+const proximityIcons: { [key: string]: IconDefinition } = {
+  WellRatedRestaurants: faUtensils,
+  GreenAreas: faTree,
+  Schools: faSchool,
+  Colleges: faUniversity,
+  Hospitals: faHospital,
+  Markets: faStore,
+};
```

Comment:
vc tem algum enum para essas chaves? Do jeito que ta escrito, parecem ser unicas. Poderia criar um enum, caso nao

```suggestion
const proximityIcons: Record<ImmobleProperties, IconDefinition> = {
  [ImmobleProperties.WellRatedRestaurants]: faUtensils,
  [ImmobleProperties.GreenAreas]: faTree,
  [ImmobleProperties.Schools]: faSchool,
  [ImmobleProperties.Colleges]: faUniversity,
  [ImmobleProperties.Hospitals]: faHospital,
  [ImmobleProperties.Markets]: faStore,
};
```